docs/advanced-usage/logging-model-events.md

```diff
@@ -200,6 +200,33 @@ class NewsItem extends Model
 Changing only `name` means only the `name` attribute will be logged in the activity, and `text` will be left out.
 
 
+
+## Logging only the specific JSON attributes sub-key
+
+If you would like to log only the changes to a specific JSON objects sub-keys. You can use the same method for logging specific columns with the difference of choosing the json key to log.
+
+```php
+use Illuminate\Database\Eloquent\Model;
+use Spatie\Activitylog\Traits\LogsActivity;
+
+class NewsItem extends Model
+{
+    use LogsActivity;
+
+    protected $fillable = ['preferences', 'name'];
+    
+    protected static $logAttributes = ['preferences->notifications->status', 'preferences->avatar_url'];
+    
+    protected $casts = [
+        'preferences' => 'collection' // casting the JSON database column
+    ];
+}
+```
+
+Changing only `preferences->notifications->status` or `preferences->avatar_url` means only the `preferences->notifications->status` or `preferences->avatar_url` attribute will be logged in the activity, and everything else `preferences` will be left out.
+
+The result in the log entry key for the attribute will be what is in the `$logAttributes`.
```

Comment:
Could you add an example output here? Because the resulting array structure is a bit different and it should be mentioned/shown that we transform the `attribute->sub->key` structure into `attribute[sub][key]`/`attribute.sub.key` structure.